velox/common/caching/SsdCache.cpp

```diff
@@ -33,6 +36,13 @@ SsdCache::SsdCache(
       numShards_(numShards),
       groupStats_(std::make_unique<FileGroupStats>()),
       executor_(executor) {
+  VELOX_CHECK(
```

Comment:
Can we add a unit test to verify if ssd cache can auto create the prefix path and if the path pre-exist, the followup file operation can succeed? Thanks!